tests/unit/utils/event_test.py

```diff
@@ -249,8 +249,12 @@ def test_event_subscriptions_cache_regex(self):
     def test_event_multiple_clients(self):
         '''Test event is received by multiple clients'''
         with eventpublisher_process():
-            me1 = event.MasterEvent(SOCK_DIR)
-            me2 = event.MasterEvent(SOCK_DIR)
+            me1 = event.MasterEvent(SOCK_DIR, listen=True)
+            me2 = event.MasterEvent(SOCK_DIR, listen=True)
+           # We need to sleep here to avoid a race condition wherein
+           # the second socket may not be connected by the time the first socket
+           # sends the event.
```

Comment:
Another set of mis-indents. Because I'm OCD.

Thanks for pointing it out, @jfindlay